profiles-sdk-examples/README.md

```diff
@@ -0,0 +1,191 @@
+    "_createdBy": "laguirre@cognitivescale.com",
+    "name": "profiles-example",
+    "description": "",
+    "image": "private-registry.dci-dev.dev-eks.insights.ai/profiles-example:latest",
+    "type": "job",
+    "command": [
+      "scuttle",
+      "python",
+      "submit_job.py"
+    ],
+    "scaleCount": 1,
+    "podSpec": "[{\"path\":\"/containers/0/imagePullPolicy\",\"value\":\"Always\"}]",
+    "jobTimeout": 0,
+    "k8sResources": [],
+    "environmentVariables": null,
+    "createdAt": "2022-07-07T17:10:14.258Z",
+    "updatedAt": "2022-07-07T17:10:14.258Z",
+    "_version": 2
+  }
+}
+cortex skills save -y templates/skill.yaml --project testi-69257
+Skill saved: {"success":true,"version":2,"message":"Skill definition profiles-example saved."}
+```
+
+Invoking the Skill:
+```
+$ make invoke
+cortex skills invoke --params-file templates/payload.json profiles-example params --project laguirre-testi-69257
+{
+  "success": true,
+  "activationId": "115d1196-408a-47fa-91c4-6f8e8a391641"
+}
+```
+
+Run `cortex agents get-activation <activation-id>` or `cortex tasks logs <task-name>` to view logs of the skill activation.
+
+## FAQ
+* How are users installing this? 
+  - See [Installation and Setup](#installation-and-setup)
+* Is this installed or configured in [Cortex Charts](https://github.com/CognitiveScale/cortex-charts)?
+  - Nope, the SDK is entirely separate from Cortex Charts (though some configuration options may be similar).
+* Is this a sort of Standalone tool? (like the Cortex CLI)
+  - Yes and no! The core of the Cortex Profiles SDK is a standalone library, that has platform (pluggable) Spark and hadoop dependencies. The SDK is its own standalone tool, but it will not stand on its own.
+* Spark has a number of levers identified in their docs, which ones are most critical for the Profiles SDK?
+  - For configuration option reference, see [here](docs/config.md). (This is missing guidance on tuning, but we can add as we iterate).
+* Is the code annotated to point out the customization areas/options? 
+  - The generated API documentation includes Spark (Cortex) Configuration options and will describe related
+    configuration properties and abilities. The initial documentation is minimal in certain areas, but bill we itreated on.
+    For a full list of configuration options see [config.md](docs/config.md)
+* Does this use the [Cortex CLI](https://www.npmjs.com/package/cortex-cli) or ~~Phoenix CLI (AMP CLI)~~?
+    - No, neither are used by nor required by the Profiles SDK. The Cortex CLI is a separate tool that would facilitate
+      developing with the Profiles SDK for generating Tokens and creating Cortex Resources. The Phoenix CLI is not
+      public facing, and the Profiles SDK will eventually overcome/replace the Phoenix CLI as it supports the same
+      functionality with more flexible configuration.
+
+## Version Compatibility
+
+<!-- TODO: Crossed out because this check shouldn't happen automatically? -->
```

Comment:
So, based on looking at the code the check does not happen automatically, but the wiring should be there so in the future we could have a smarter version compatibility check.

I can see how the table is possibly confusing because users don't need to specify the Spark/Hadoop version when running these examples (because its using our prepackaged platform dependencies). Users are not forced to use our platform dependencies, and they can update that in the build script to use other versions.

I guess its unnecessary since that probably won't be happen soon, but I think there should be a table somewhere because the SDK is only compatible with certain versions of the dependencies. I'll comment out this entire section for now..